joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForDeclarationsCreator.scala

```diff
@@ -52,19 +56,27 @@ trait AstForDeclarationsCreator(implicit withSchemaValidation: ValidationMode) {
     memberNodes.map(Ast(_).withChildren(astForModifiers(fieldDecl)))
   }
 
+  protected def astForLocalDeclarationStatement(localDecl: DotNetNodeInfo): Seq[Ast] = {
+    astForVariableDeclaration(createDotNetNodeInfo(localDecl.json(ParserKeys.Declaration)))
+  }
   protected def astForVariableDeclaration(varDecl: DotNetNodeInfo): Seq[Ast] = {
     val typeFullName = nodeTypeFullName(varDecl)
     varDecl
       .json(ParserKeys.Variables)
       .arr
       .map(createDotNetNodeInfo)
-      .map(astForVariableDeclarator(_, typeFullName))
+      .flatMap { nodeInfo =>
```

Comment:
Could this not just be
```suggestion
      .flatMap(astForVariableDeclarator(_, typeFullName))
```